src/data/template/static/js/control/html.js

```diff
@@ -1,12 +0,0 @@
-HTMLControl.prototype = new BaseControl("HTMLControl");
```

Comment:
I think it should be fine to remove `HTMLControl`. It's not referenced anywhere else. If you think it should stay, let me know.